### ISSUE TITLE
ckan container crashes at startup

### ISSUE THREAD

**Issue by mmartain** [ckan/ckan#5243]
**Note:** Are you submitting a security related issue that could be a potential vulnerability? Please send it to security@ckan.org instead.

### CKAN Version if known (or site URL)
2.9.0a0

### Please describe the expected behaviour
Ckan up and running with 3 docker commands.
$ docker run -d --name db ckan/postgresql
$ docker run -d --name solr ckan/solr
$ docker run -d -p 80:80 --link db:db --link solr:solr ckan/ckan

### Please describe the actual behaviour
Ckan container crashes:
db:5432 - accepting connections
Distribution already installed:
  ckan 2.9.0a0 from /usr/lib/ckan/venv/src/ckan
Creating /etc/ckan/production.ini
Now you should edit the config files
  /etc/ckan/production.ini
ERROR: no CKAN_SQLALCHEMY_URL specified in docker-compose.yml

### What steps can be taken to reproduce the issue? 
$ docker run -d --name db ckan/postgresql
$ docker run -d --name solr ckan/solr
$ docker run -d -p 80:80 --link db:db --link solr:solr ckan/ckan

**Comment by kowh-ai** [comment]
@mmartain I'll take a look at this today...had network problems here the last couple of days - all sorted now (hopefully)

**Comment by kowh-ai** [comment]
Yes - I'm getting the same issue on Ubuntu 16.04. I'm getting a different error when installing via docker-compose (ImportError: No module named configparser)...CKAN master still a way to...Q: Were you wanting to run the docker containers individually (rather than via docker-compose) because of the docker-compose problems?

**Comment by kowh-ai** [comment]
I presume you saw this (similar issue from a couple of years ago) ? https://stackoverflow.com/questions/47550402/no-ckan-sqlalchemy-url-specified-when-installing-ckan-on-docker

**Comment by kowh-ai** [comment]
@mmartain - PR 5381 mentioned above by Mark fixes the problem
